### PR TITLE
ATLButtonLabel

### PR DESCRIPTION
**Pull request by Isasprengel**

### CODE REVIEW

File: atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        button?.textColorProp = style.labelFontColor
+        
+        // {fixed}
+        button?.titleLabel?.numberOfLines = 2
+        button?.titleLabel?.sizeToFit()
+        button?.borderColorProp = .clear
+    }
+    
+    func updateState() {
+//        button?.titleLabel?.text = viewState.text?.capitalized
```
Comment (kyonaganawa) [inline atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift]: Can we remove this comment?

File: atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        marginRightSpacing?.constant = style.marginRightSpacing
+        marginTopSpacing?.constant = style.marginTopSpacing
+        marginBottomSpacing?.constant = style.marginBottomSpacing
+    }
+    
+    func updateLabel() {
+        button?.titleLabel?.font = style.labelFont
+        button?.textColorProp = style.labelFontColor
+        
+        // {fixed}
```
Comment (kyonaganawa) [inline atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift]: Can we remove this comment? If it's something important, leave a more clear message

File: atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    
+    public var viewState = ATLLabelButtonStateProps() {
+        didSet {
+            updateState()
+        }
+    }
+
+    //MARK: Actions
+    @IBAction func buttonClicked() {
+       //TODO click
```
Comment (kyonaganawa) [inline atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift]: We can call a delegate function here

File: atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift (hunk 1 above)
Comment (Isasprengel) [reply atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift]: Check!

File: atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift (hunk 2 above)
Comment (Isasprengel) [reply atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift]: Check!

File: atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift (hunk 3 above)
Comment (Isasprengel) [reply atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift]: Check!

File: atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+            return ATLTokens.mainTheme.colors.primaryColor
+        case .complementary:
+            return ATLTokens.mainTheme.colors.complementaryColor
+        }
+    }
+}
+
+//MARK: Delegate
+public protocol ATLLabelButtonDelegate: class {
+    func buttokClick()
```
Comment (kyonaganawa) [inline atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift]: There's a typo here. Rudi proposed that we use onButtonClick() or similar names for those actions. What do you think?

Comment (Isasprengel) [reply atlantica-ios/Classes/Molecules/ATLLabelButton/ATLLabelButton.swift]: Check!

**Pull request merged**